test/blocks/merch-card/offer-selection.test.js

```diff
@@ -0,0 +1,70 @@
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+
+const { default: initCard } = await import('../../../libs/blocks/merch-card/merch-card.js');
+
+describe('Offer Selection', () => {
+  before(async () => {
+    document.body.innerHTML = await readFile({ path: './mocks/acrobat-card.html' });
+    document.body.innerHTML = await readFile({ path: './mocks/acrobat-card.html' });
+    await initCard(document.querySelector('.acrobat'));
+    await new Promise((resolve) => { setTimeout(resolve, 100); });
```

Comment:
you can declare a delay function and re-use it.
```
const delay = (duration = 100) => new Promise((resolve) => { setTimeout(resolve, duration); });
...
await delay();
```

Reply:
yep, did that